### PR TITLE
linux: alert tuning for k4spreader, injector, medusa, Sliver

### LINKED ISSUE #324: Add detection for Medusa preload
https://github.com/ldpreload/Medusa

Mentioned here: https://cloud.google.com/blog/topics/threat-intelligence/uncovering-unc3886-espionage-operations/

### PR DESCRIPTION
**Pull request by tstromberg**
fixes #324 #325 #327

unrelated change: I changed the chrome testdata file from .md to .simple to be easier to reason about.

### CODE REVIEW

File: rules/evasion/fake-process.yara [hunk 1]
```diff
@@ -42,3 +43,17 @@ rule fake_systemd : critical {
...
+rule fake_process_names : high {
+	meta:
+		description = "mentions known fake process names"
+	strings:
+		$kdevchecker = "kdevchecker" fullword
+		$kworkerr = "kworkerr" fullword
+		$ksoftriqd = "ksoftriqd" fullword
+		$kdevtmpfsi = "kdevtmpfsi" fullword
+		$kthreaddk = "kthreaddk" fullword
+		$deamon = "deamon" fullword
```
Comment (egibs) [inline rules/evasion/fake-process.yara:56]: Should this be `daemon` in both cases?

Comment (tstromberg) [reply rules/evasion/fake-process.yara:56]: Nope - this is an intentional misspelling based on what I found in malware samples.

**Comment by tstromberg** [comment]
@egibs - this could use another look. thanks!

File: test_data/javascript/clean/frequency_lists.js.simple [hunk 2]
```diff
@@ -1,10 +1,12 @@
-# javascript/clean/frequency_lists.js: high
+# javascript/clean/frequency_lists.js: critical
```
Comment (egibs) [inline test_data/javascript/clean/frequency_lists.js.simple:1]: Should we allow any clean samples to register as critical?

Comment (tstromberg) [reply test_data/javascript/clean/frequency_lists.js.simple:1]: In general, I would say no - but until we can define filetype rules, I think it's OK to present the fact that the rule is overly broad.

What I would like to define for `combo/degrader/infection` is: `filetypes = "!js"`

**Review by egibs — APPROVED** [review]

**Pull request merged**